packages/cli/test/helpers/read-output-stream.ts

```diff
@@ -0,0 +1,23 @@
+import { MockClient } from '../mocks/client';
+
+export function readOutputStream(
```

Comment:
The need for this was created by removing `outputBuffer` in: https://github.com/vercel/vercel/pull/8052

I think we should add `outputBuffer` (or something like it) back in. I created a PR to do just that, which would remove this file. https://github.com/vercel/vercel/pull/8108